conans/client/command.py

```diff
@@ -61,8 +61,8 @@ def __call__(self, parser, namespace, values, option_string=None):
                       "relative path to current directory can also be specified")
 _INSTALL_FOLDER_HELP = ("Directory containing the conaninfo.txt and conanbuildinfo.txt files "
                         "(from previous 'conan install'). Defaulted to --build-folder")
-_KEEP_SOURCE_HELP = ("Do not remove the source folder in local cache. Use this for testing purposes"
-                     " only")
+_KEEP_SOURCE_HELP = ("Do not remove the source folder in local cache, even if the recipe is outdated. "
```

Comment:
To be accurate, the recipe is not outdated, but a new recipe makes the binaries outdated. I would say something like: ``even if the recipe has changes``.

Reply:
Would `even if the recipe changed` be ok ?

Reply:
Yes, sure!